index.html

```diff
@@ -29,7 +27,108 @@ <h1>I'm Egbe Bless Ashuakie!</h1>
+                    <li><p>javaScript</p></li>
+                </ul>
+                <button type="button" class="btn see-projects">See Projects</button>
+            </div>
+            <div class="card">
+                <img src="images/first_card_image.png" alt="">
+                <h2>Tonic</h2>
+                <ul class="skills">
+                    <li class="canopy"><p>Canopy</p></li>
+                    <li><img src="images/small-circle.png" alt=""></li>
+                    <li><p>Back End Dev</p></li>
+                    <li><img src="images/small-circle.png" alt=""></li>
+                    <li><p>2015</p></li>
+                </ul>
+                <p class="daily-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="langs">
+                    <li><p>html</p></li>
+                    <li><p>css</p></li>
+                    <li><p>javaScript</p></li>
+                </ul>
+                <button class="btn see-projects" type="button">See Projects</button>
+            </div>
+            <div class="card">
+                <img src="images/first_card_image.png" alt="">
+                <h2>Tonic</h2>
+                <ul class="skills">
+                    <li class="canopy"><p>Canopy</p></li>
+                    <li><img src="images/small-circle.png" alt=""></li>
+                    <li><p>Back End Dev</p></li>
+                    <li><img src="images/small-circle.png" alt=""></li>
+                    <li><p>2015</p></li>
+                </ul>
+                <p class="daily-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="langs">
+                    <li><p>html</p></li>
+                    <li><p>css</p></li>
+                    <li><p>javaScript</p></li>
+                </ul>
+                <button class="btn see-projects" type="button">See Projects</button>
+            </div>
+            <div class="card">
+                <img src="images/first_card_image.png" alt="">
+                <h2>Tonic</h2>
+                <ul class="skills">
+                    <li class="canopy"><p>Canopy</p></li>
+                    <li><img src="images/small-circle.png" alt=""></li>
+                    <li><p>Back End Dev</p></li>
+                    <li><img src="images/small-circle.png" alt=""></li>
+                    <li><p>2015</p></li>
+                </ul>
+                <p class="daily-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="langs">
+                    <li><p>html</p></li>
+                    <li><p>css</p></li>
+                    <li><p>javaScript</p></li>
+                </ul>
+                <button class="btn see-projects" type="button">See Projects</button>
+            </div>
+        </div>
+        <div class="about_myself_section">
```

Comment:
- [ ] [OPTIONAL] Please wrap each section in `section` tag instead of `div` tags. That would make the code semantically correct.